contracts/utility-token/UtilityToken.sol

```diff
@@ -51,6 +49,63 @@ contract UtilityToken is ERC20Token {
 
     /* External Functions */
 
+    /**
+     * @notice Sets up the symbol, name, decimals, totalSupply, value token
+     *         and the consensusCogateway address.
+     *
+     * @param _symbol Symbol of token.
+     * @param _name Name of token.
+     * @param _decimals Decimal of token.
+     * @param _totalTokenSupply Total token supply.
+     * @param _consensusCogateway ConsensusCogateway contract address.
+     * @param _valueToken Address of value token contract.
+     *
+     * \pre Setup is not already called.
```

Comment:
` * \pre This function can be called only once.`
or
` * \pre Setup function can be called only once.`

Reply:
Done